email_template/src/templates/magic_link.html

```diff
@@ -1,131 +1,131 @@
-                </p>
-              </td>
-            </tr>
-            <tr>
-              <td>
-                <x-button
-                  align="center"
-                  class="hover:opacity-90 cursor-pointer !bg-[#9327ff] m-0 rounded-2xl"
-                  href="@{{ .ConfirmationURL }}"
-                >
-                  <div class="font-bold text-[16px]">Login to AppFlowy</div>
-                </x-button>
-              </td>
-            </tr>
-          </table
-        </td>
-      </tr>
-      <tr>
-        <td>
-          <div class="mx-auto leading-4.5 text-base text-center">
-            If the link does not work, you can use the verification code
-            directly:
-          </div>
-        </td>
-      </tr>
-      <tr>
-        <td class="text-center">
-          <p class="bg-gray-200 p-2 rounded text-center text-black text-lg font-bold inline-block m-0">@{{ .Token }}</p>
-        </td>
-      </tr>
-      <tr>
-        <td>
-          <x-divider space-x="10%" />
-        </td>
-      </tr>
-      <tr>
-        <td>
-          <p class="m-0 text-sm text-gray-600 font-medium text-center">
-            Bring projects, knowledge, and teams together with the power of AI.
-          </p>
-
-          <p class="cursor-default text-center">
-            <a
-              href="https://twitter.com/appflowy"
-              class="[text-decoration:none] mr-4"
-            >
-              <img
-                src="{{ cdnBaseUrl }}images/twitter.png"
-                width="20"
-                alt="Maizzle"
-              />
-            </a>
-            <a
-              href="https://www.reddit.com/r/AppFlowy"
-              class="[text-decoration:none] mr-4"
-            >
-              <img
-                src="{{ cdnBaseUrl }}images/reddit.png"
-                width="20"
-                alt="Maizzle"
-              />
-            </a>
-            <a
-              href="https://github.com/AppFlowy-IO/AppFlowy"
-              class="[text-decoration:none] mr-4"
-            >
-              <img
-                src="{{ cdnBaseUrl }}images/github.png"
-                width="20"
-                alt="Maizzle"
-              />
-            </a>
-            <a
-              href="https://discord.gg/9Q2xaN37tV"
-              class="[text-decoration:none] mr-4"
-            >
-              <img
-                src="{{ cdnBaseUrl }}images/discord.png"
-                width="20"
-                alt="Maizzle"
-              />
-            </a>
-          </p>
-          <p class="m-0 text-sm text-gray-600 font-medium text-center">
-            Copyright © 2025, AppFlowy Inc.
-          </p>
-          <p class="text-sm text-gray-600 font-medium text-center">
-            Need Help? <a href="mailto:support@appflowy.io" class="text-gray-600">support@appflowy.io</a>
-          </p>
-        </td>
-      </tr>
-    </table>
+  <div style="display: none">
```

Comment:
Is this div added for debugging/dev purposes? Or do we need this div for other reasons?